abr_control/controllers/path_planners/linear.py

```diff
@@ -16,8 +16,8 @@ class Linear(PathPlanner):
     def __init__(self, robot_config):
         super(Linear, self).__init__(robot_config)
 
-    def generate_path(self, state, target, n_timesteps=200,
-                 dt=0.001, plot=False):
+    def generate_path(self, state, target, n_timesteps=None,
```

Comment:
updated linear path planner with constant vel

Reply:
we merged this PR in master